src/cdk/a11y/key-manager/list-key-manager.spec.ts

```diff
@@ -818,6 +818,18 @@ describe('Key managers', () => {
         expect(keyManager.activeItem).toBe(itemList.items[1]);
       }));
 
+      it('should expose whether the user is currently typing', fakeAsync(() => {
+        expect(keyManager.isTyping()).toBe(false);
+
+        keyManager.onKeydown(createKeyboardEvent('keydown', 79, 'o')); // types "o"
```

Comment:
out of curiosity: why not using `cdk/keycodes` for the `O` keycode? (for consistency)

Reply:
Nevermind.. I see other tests in this file hardcode the code too.

Reply:
I think most of the tests in this file were written before we added as many constants to the CDK.